specification/columns/servicesubcategory.md

```diff
@@ -0,0 +1,103 @@
+| Compute                   | Containers                        | Automated management of containerized applications.                                                                                                                                                    |
+|                           | End User Computing                | Desktop virtualization, mobile computing, and self-service analytics.                                                                                                                                  |
+|                           | Quantum                           | Resources and simulators that leverage the principles of quantum mechanics.                                                                                                                            |
+|                           | Serverless Compute                | Facilitates building and running applications without managing servers.                                                                                                                                |
+|                           | Virtual Machines                  | Computing environments created by abstracting resources from physical hardware.                                                                                                                        |
+|                           | Other (Compute)                   | Compute services that do not fall into one of the defined subcategories.                                                                                                                               |
+| Databases                 | Caching                           | Frequently accessed data housed in a temporary storage area for quick retrieval.                                                                                                                       |
+|                           | Data Warehouse                    | Centralized repository that stores large volumes of structured data to facilitate analytical use cases.                                                                                                |
+|                           | Relational                        | Data organized into tables with rows and columns to facilitate transactional use cases.                                                                                                                |
+|                           | NoSQL                             | Unstructured or semi-structured data to facilitate use cases where data models change frequently.                                                                                                      |
+|                           | Ledger                            | Transparent, immutable, and cryptographically verifiable transaction log used in applications requiring a secure and auditable record of transactions.                                                 |
```

Comment:
```suggestion
|                           | Ledger                            | Immutable and transparent databases for tamper-evident and cryptographically secure records of transactions and data                                                 |
```

Reply:
`Tamper-evident`?  I've not heard this term, and Googling it suggests `tamper-proof` might be the better term.  How about:

```suggestion
|                           | Ledger                            | Immutable and transparent databases to record tamper-proof and cryptographically secure transactions.                                                 |
```